### PR TITLE
ansible: rhel7-s390x manual ninja/v8 build step

### PR DESCRIPTION
**Pull request by sam-github**
Needed for V8 jobs. Description is incomplete wrt. how to build them
initially, but better than the previous state of no docs at all.

@AshCripps is going to look at ansiblizing this, and the existence of documented manual steps will help understand what needs to be ansibled.

### CODE REVIEW

**Comment by AshCripps** [comment]
https://github.com/nodejs/build/blob/fbcbd6ce637ea75e895dd1e788ce604303c1fb63/ansible/aix61-standalone/ansible-playbook.yaml#L19-L21

Found an example of how it can be done - just need to get the files uploaded to ci.nodejs.org and I can pr in the ansible for it.

File: ansible/MANUAL_STEPS.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Manual steps required to run ansible on machines
 
+## RHEL7-S390X
+1. V8 build-tools
+
+Copy `/home/iojs/build-tools/{gn,ninja}` from any other RHEL7-S390X host that
+has them to the newly provisioned one.
+
+If building from scratch, gn is built from  https://gn.googlesource.com/gn,
+and ninja is built from https://github.com/nodejs/node-v8.git
```
Comment (AshCripps) [inline ansible/MANUAL_STEPS.md]: is this the right link for ninja?

Comment (AshCripps) [inline ansible/MANUAL_STEPS.md]: Is this the right link for ninja?

Comment (sam-github) [reply ansible/MANUAL_STEPS.md]: Hm, looks odd, I think it was what the remotes were setup on the build repos I found.

File: ansible/MANUAL_STEPS.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Manual steps required to run ansible on machines
 
+## RHEL7-S390X
+1. V8 build-tools
+
+Copy `/home/iojs/build-tools/{gn,ninja}` from any other RHEL7-S390X host that
+has them to the newly provisioned one.
```
Comment (AshCripps) [inline ansible/MANUAL_STEPS.md]: before copying across check the gn binary is valid by running `gn --version` which should return a version number and not `UNKNOWN`. Also I think the link for ninja is wrong.

Comment (sam-github) [reply ansible/MANUAL_STEPS.md]: ok. current state needs fixing, I guess?
```
core/build (add-aix71-testing $% u=) % parallel-ssh -h hosts/rhel7-s390x -i "sudo /home/iojs/build-tools/gn --version" 
[1] 11:33:18 [FAILURE] test-ibm-rhel7-s390x-3 Exited with error code 1
Stderr: sudo: /home/iojs/build-tools/gn: command not found
[2] 11:33:18 [SUCCESS] test-ibm-rhel7-s390x-2
1620 (570aaed)
[3] 11:33:18 [SUCCESS] test-ibm-rhel7-s390x-4
UNKNOWN
[4] 11:33:19 [SUCCESS] test-ibm-rhel7-s390x-1
UNKNOWN
core/build (add-aix71-testing $% u=) % parallel-ssh -h hosts/rhel72-s390x -i "/home/iojs/build-tools/gn --version" 
[1] 11:33:32 [SUCCESS] test-linuxonecc-rhel72-s390x-2
UNKNOWN
[2] 11:33:32 [FAILURE] test-linuxonecc-rhel72-s390x-3 Exited with error code 127
Stderr: bash: /home/iojs/build-tools/gn: No such file or directory
[3] 11:33:33 [SUCCESS] test-linuxonecc-rhel72-s390x-1
1620 (570aaed)
```

**Pull request merged**